intel-retail/automated-self-checkout
# feat: convert ovms-client from docker into binary executable and updated ovms launch script

- Converting the existing ovms-client docker into a binary executable and remove ovms-client docker container related stuff.
- rename docker-run.sh to run.sh
- rename docker-run-dlstreamer.sh to run-dlstreamer.sh
- rename docker-run-opencv-ovms.sh to run-ovms.sh
- reanme opencv-ovms workload parameter to ovms
- ovms-client docker is now called profile-launcher and it is binary executable

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- build profile launcher binary executable: `make build-profile-launcher`
- you can try to run with the **test** profile: 
```console
DETECTION_THRESHOLD=0.77 PIPELINE_PROFILE="test" sudo -E ./run.sh --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms --platform core
```
and you should be able to observe that it runs the test script launched by profile-launcher and outputs including the environment overrides
- TBD:   
1. rename top level script (something else instead of docker-run.sh) **DONE**
1. remove or delete Dockerfile for .ovms-client  **DONE**
1. fix the ci trivy scanning related to ovms-client docker image  **DONE**


Test it with `instance_segmentation` profile and it works too:
```console
1. $ ./camera_simulator/camera_simulator.sh
2. $ PIPELINE_PROFILE="segmentation_instance" RENDER_MODE=1 sudo -E ./run.sh --workload ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
```

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
